core/Xargo.toml

```diff
@@ -0,0 +1,4 @@
+std = { features = ["jemalloc"] }
```

Comment:
What do

Reply:
It sets the memory allocator to `jemalloc`